apps/src/templates/sectionsRefresh/CurriculumQuickAssignTopRow.jsx

```diff
@@ -12,84 +13,96 @@ export const MARKETING_AUDIENCE = {
+        updateMarketingAudience('');
+      } else {
+        updateMarketingAudience(newAudience);
+      }
+    },
+    [marketingAudience, updateMarketingAudience]
+  );
 
   return (
     <div className={moduleStyles.buttonRow}>
       <div className={moduleStyles.buttonsInRow}>
-        <Button
-          id={'uitest-elementary-button'}
-          className={moduleStyles.buttonStyle}
+        <MarketingAudienceButton
+          selectedMarketingAudience={marketingAudience}
+          audience={MARKETING_AUDIENCE.ELEMENTARY}
+          determineMarketingAudience={determineMarketingAudience}
           text={i18n.courseBlocksGradeBandsElementary()}
-          size={Button.ButtonSize.large}
-          icon={marketingAudienceIcon(MARKETING_AUDIENCE.ELEMENTARY)}
-          onClick={e => {
-            e.preventDefault();
-            determineMarketingAudience(MARKETING_AUDIENCE.ELEMENTARY);
-          }}
         />
-        <Button
-          id={'uitest-middle-button'}
-          className={moduleStyles.buttonStyle}
+        <MarketingAudienceButton
+          selectedMarketingAudience={marketingAudience}
+          audience={MARKETING_AUDIENCE.MIDDLE}
+          determineMarketingAudience={determineMarketingAudience}
           text={i18n.courseBlocksGradeBandsMiddle()}
-          size={Button.ButtonSize.large}
-          icon={marketingAudienceIcon(MARKETING_AUDIENCE.MIDDLE)}
-          onClick={e => {
-            e.preventDefault();
-            determineMarketingAudience(MARKETING_AUDIENCE.MIDDLE);
-          }}
         />
-        <Button
-          id={'uitest-high-button'}
-          className={moduleStyles.buttonStyle}
+        <MarketingAudienceButton
+          selectedMarketingAudience={marketingAudience}
+          audience={MARKETING_AUDIENCE.HIGH}
+          determineMarketingAudience={determineMarketingAudience}
           text={i18n.courseBlocksGradeBandsHigh()}
-          size={Button.ButtonSize.large}
-          icon={marketingAudienceIcon(MARKETING_AUDIENCE.HIGH)}
-          onClick={e => {
-            e.preventDefault();
-            determineMarketingAudience(MARKETING_AUDIENCE.HIGH);
-          }}
         />
-        <Button
-          id={'uitest-hoc-button'}
-          className={moduleStyles.buttonStyle}
+        <MarketingAudienceButton
+          selectedMarketingAudience={marketingAudience}
+          audience={MARKETING_AUDIENCE.HOC}
+          determineMarketingAudience={determineMarketingAudience}
           text={i18n.teacherCourseHoc()}
-          size={Button.ButtonSize.large}
-          icon={marketingAudienceIcon(MARKETING_AUDIENCE.HOC)}
-          onClick={e => {
-            e.preventDefault();
-            determineMarketingAudience(MARKETING_AUDIENCE.HOC);
-          }}
         />
+
         {showPlOfferings && (
-          <Button
-            id={'uitest-pl-button'}
-            className={moduleStyles.buttonStyle}
+          <MarketingAudienceButton
```

Comment:
I love how clean the render is after the refactor! Thank you for that update!

Reply:
I'm guessing you meant to tag Molly Peredo? I can't find her on github to tag in.

Reply:
Omg, I'm sorry Molly! Yeah, I've meant Molly Peredo. I'll send her a link in slack!